front/src/applications/stdcm/components/SimulationReportSheet.tsx

```diff
@@ -0,0 +1,272 @@
+                <Text style={styles.convoyInfoTitles}>{t('maxLength')}</Text>
+                <Text style={styles.convoyInfoData}>
+                  {rollingStockData?.length} {t('meters')}
+                </Text>
+              </View>
+            </View>
+          </View>
+          <View style={styles.route}>
+            <Text style={styles.routeTitle}> {t('requestedRoute')}</Text>
+            <View style={styles.fromBanner}>
+              <View style={styles.fromBox}>
+                <Text style={styles.from}>{t('from')}</Text>
+              </View>
+              <Text style={styles.fromNumber}>n°123456</Text>
+            </View>
+            <View style={styles.stopTableContainer}>
+              <Table style={styles.stopTable}>
+                <TH style={styles.stopTableTH}>
+                  <TD aria-label="line-count" />
+                  <TD>{t('operationalPoint')}</TD>
+                  <TD>{t('code')}</TD>
+                  <TD>{t('endStop')}</TD>
+                  <TD>{t('startStop')}</TD>
+                  <TD>{t('motif')}</TD>
+                </TH>
+                {stdcmData?.simulation.base.stops.map(
+                  (stop, index) =>
+                    (index === 0 || stop.duration > 0) && (
+                      <TR key={index} style={styles.stopTableTbody}>
+                        <TD style={styles.stopTableIndexColumn}>{index + 1}</TD>
+                        <TD style={styles.stopTablePRColumn}>{stop.name || 'Unknown'}</TD>
+                        <TD style={styles.stopTableCHColumn}>{stop.ch}</TD>
+                        <TD style={styles.stopTableItalicColumn}>
+                          {index === stdcmData.simulation.base.stops.length - 1
+                            ? getStopTime(stop.time)
+                            : ''}
+                        </TD>
+                        <TD style={styles.stopTableStartColumn}>
+                          {index === 0 ? getStopTime(stop.time) : ''}
+                        </TD>
+                        <TD style={styles.stopTableItalicColumn}>
+                          {index === 0 || index === stdcmData.simulation.base.stops.length - 1
+                            ? t('serviceStop')
+                            : ''}
+                        </TD>
+                      </TR>
+                    )
+                )}
+              </Table>
+            </View>
+            <View style={styles.forBanner}>
+              <Text style={styles.forNumber}>n°987654</Text>
+              <View style={styles.forBox}>
+                <Text style={styles.for}>{t('for')}</Text>
+              </View>
+            </View>
+          </View>
+        </View>
+        <View style={styles.simulation}>
+          <View style={styles.simulationContainer}>
+            <Text style={styles.simulationUppercase}>{t('simulation')}</Text>
+            <Text style={styles.simulationLength}>
+              {Math.round((stdcmData?.path.length ?? 0) / 1000)}
```

Comment:
+ Use a string `${...} km`
```suggestion
              {Math.round(stdcmData.path.length / 1000)}
```

Reply:
done